src/Domain/Entity/Image.php

```diff
@@ -63,7 +65,12 @@ public function getLongestSynopsis(): string
     public function getUrl($width, $height = 'n'): string
     {
         $recipe = $width . 'x' . $height;
-        return 'https://ichef.bbci.co.uk/images/ic/' . $recipe . '/' . $this->filename;
+        return self::BASE_URL . '/' . $recipe . '/' . $this->filename;
+    }
+
+    public function getRecipeUrl(): string
+    {
+        return self::BASE_URL . '/$recipe/' . $this->filename;
```

Comment:
This looks similar but different to the one above. Is it definitely correct?

Reply:
It's very similar, I wasn't sure whether to put this here or not really, but I thought it's better to have this stuff defined in one place. 
Some stuff needs a URL where the dimensions can be dynamically relaced (SMP)

I could just make the base url constant public and add a getter for the filename otherwise. Any opinions?

Reply:
FML. Maybe a note that it is correct to be a string and isn't supposed to be a variable?